web/cypress/integration/countrypanel.spec.js

```diff
@@ -1,9 +1,4 @@
 describe('Country Panel', () => {
-  it('asserts countryPanel contains carbon intensity when available', () => {
```

Comment:
Maybe worth adding a TODO here so we don't forget to reintroduce a test later

Reply:
Hmm I don't think that's necessary I just added the test initially yesterday, and I am not sure we'd want to handle the test like this